Jenkinsfile

```diff
@@ -11,10 +11,10 @@ def images = [
     [image: "centos:8",                       arches: ["amd64", "aarch64"]],          // Note: armhf (arm32) images are currently not available on Docker Hub
     [image: "debian:stretch",                 arches: ["amd64", "aarch64", "armhf"]], // Debian 9  (EOL: June, 2022)
     [image: "debian:buster",                  arches: ["amd64", "aarch64", "armhf"]], // Debian 10 (EOL: 2024)
-    [image: "fedora:29",                      arches: ["amd64", "aarch64"]],
     [image: "fedora:30",                      arches: ["amd64", "aarch64"]],
     [image: "fedora:31",                      arches: ["amd64", "aarch64"]],
-    [image: "fedora:latest",                  arches: ["amd64"]],
```

Comment:
`:latest` points to the current release, so we were testing Fedora 31 twice; changed to fedora "rawhide", which is always the upcoming (dev) release